server/controllers/games.js

```diff
@@ -0,0 +1,73 @@
+const Game = require('../models').Game;
+
+
+module.exports= {
+    create(req, res) {
+        return Game
+            .create({
+                game_number: req.body.game_number,
+                sr_change: req.body.sr_change,
+                rank: req.body.rank,
+                streak: req.body.streak,
```

Comment:
should streak be stored in the database?

It at least doesn't seem like a thing that belongs to a Game object